# Description:
#   managed_ecu: <- Fixed, Don't Change.
#     ecu_name: <- Any number of units can be added
#       state: {EcuState topic name}
#       prepare: {PrepareShutdown service name}
#       execute: {Executeshutdown service name}
#       skip_shutdown: {when true, the manager do not execute shutdown for this unit}
#
# Note:
# default values are:
#   state: /{ecu_name}/get/ecu_state
#   prepare: /api/{ecu_name}/prepare_shutdown
#   execute: /api/{ecu_name}/execute_shutdown
#   primary: false
#   skip_shutdown: false
---
/**:
  ros__parameters:
    update_rate: 1.0
    startup_timeout: 60.0
    preparation_timeout: 10.0
    server_port_: 10000
    server_timeout: 1
    publisher_port: 10001
    managed_ecu:
      autoware_ecu:
        state: /autoware_ecu/get/ecu_state
        prepare: /api/autoware_ecu/prepare_shutdown
        execute: /api/autoware_ecu/execute_shutdown
        primary: true
      mot:
        skip_shutdown: true 
      logging: default
